documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -22,12 +22,14 @@
 * npm version 1 package-lock.json and npm-shrinkwrap.json file parsing has been restored.
 * The `detect.project.codelocation.unmap` property has been deprecated.
 * Changed [detect_product_long]'s JAR signing authority from Synopsys, Inc. to Black Duck Software, Inc.
+* Yarn 4 support does not require use of --detect.yarn.ignore.all.workspaces flag. Thus, properties like detect.yarn.dependency.types.excluded=NON_PRODUCTION are available for Yarn 4 projects.
```

Comment:
A little wording re-org suggestion:
* Use of the --detect.yarn.ignore.all.workspaces flag is not required for Yarn 4 projects, thus configuration parameters such as detect.yarn.dependency.types.excluded=NON_PRODUCTION can be employed.

Reply:
Done, thanks!